website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -8,28 +8,18 @@ keywords:
 
 <Changelog>
 
+* **v1.3.0**: Metrics are considered 
```

Comment:
@callum-mcdata what does this mean? or are we trying to say Metrics are operational?

Reply:
:sweat: woopsies! I mean to say that they are no longer experimental